proto/ibc/lightclients/solomachine/v1/solomachine.proto

```diff
@@ -143,7 +143,7 @@ message PacketAcknowledgementData {
   bytes acknowledgement = 2;
 }
 
-// PacketAcknowledgementAbsenceSignBytes returns the SignBytes data for
+// PacketReceiptAbsenceSignBytes returns the SignBytes data for
```

Comment:
This comment doesn't match the message name below.

Reply:
the other comments that are also mismatched will be fixed in #7394